source/client/python/api-v0.1/api/connector.py

```diff
@@ -32,27 +31,14 @@
 
 
 def get_safe_session_id():
-    """this function returns a safe uuid.
-
-    Args:
+    """
+    This function returns a safe uuid.
 
     Returns:
       str: a safe session id
 
     """
-    session_id = uuid.uuid1()
-    # if (session_id.is_safe != uuid.SafeUUID.safe):
-    #     print("Need to create a second UUID")
-    #     session_id = uuid.uuid1()
-    # if (session_id.is_safe != uuid.SafeUUID.safe):
-    #     raise Exception('Cannot produce a safe unique ID')
-
-    key = "{}-{}".format(
-        session_id,
-        generate_random_logical_partition_name()
-    )
-
-    return key
+    return str(uuid.uuid1())
```

Comment:
Any reason we are not using uuid4 ? Or better said, any reason we want dependencies with host and time generated sequences ?